examples/workspace/WORKSPACE.bazel

```
@@ -0,0 +1,56 @@
+workspace(name = "workspace_example")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive", "http_file")
+
+http_archive(
+    name = "aspect_rules_js",
+    sha256 = "d66f8abf914a0454a69181b7b17acaae56d7b0e2784cb26b40cb3273c4d836d1",
+    strip_prefix = "rules_js-2.2.0",
+    url = "https://github.com/aspect-build/rules_js/releases/download/v2.2.0/rules_js-v2.2.0.tar.gz",
+)
+
+load("@aspect_rules_js//js:repositories.bzl", "rules_js_dependencies")
+
+rules_js_dependencies()
+
+load("@aspect_rules_js//js:toolchains.bzl", "DEFAULT_NODE_VERSION", "rules_js_register_toolchains")
+
+rules_js_register_toolchains(node_version = DEFAULT_NODE_VERSION)
+
+load("@aspect_rules_js//npm:repositories.bzl", "npm_translate_lock")
+
+npm_translate_lock(
+    name = "npm",
+    pnpm_lock = "//:pnpm-lock.yaml",
+)
+
+load("@npm//:repositories.bzl", "npm_repositories")
+
+npm_repositories()
+
+local_repository(
+    name = "rules_playwright",
+    path = "../..",
+)
+
+http_file(
+    name = "browsers",
+    url = "https://unpkg.com/playwright-core@1.51.0/browsers.json",
+)
+
+load("@rules_playwright//playwright:repositories.bzl", "define_browsers", "playwright_repository")
+
+define_browsers(
+    name = "browsers_index",
+    browsers_json = "@browsers//file:downloaded",
+)
+
+load("@browsers_index//:browsers.bzl", "fetch_browsers")
+
+fetch_browsers()
+
+playwright_repository(
+    name = "playwright",
+    playwright_version = "1.51.0",
```

I notice that this rule internally also downloads `"https://unpkg.com/playwright-core@1.51.0/browsers.json",`

I wonder if we could hide this file as an implementation detail.

Otherwise, I think we should add browsers_json to avoid downloading the same thing twice.
```suggestion
    playwright_version = "1.51.0",
    browsers_json = "@browsers//file:downloaded",
```